### ISSUE TITLE
[4.0] Save Button in edit views: Dropdowns with overflow

### ISSUE THREAD

**Issue by chmst** [joomla/joomla-cms#22012]
### Steps to reproduce the issue
Edit an article and open the Save-Dropdown.

### Expected result
The dropdown is in a line with the left border of the save button.
The Dropdown contains the whole text of the options

### Actual result
If your language is English, The dropdown seems to be offset to the left side.
If your language is german or another language with longer words, the text overflows the dropdown

![screen shot 2018-09-04 at 21 45 04](https://issues.joomla.org/uploads/1/1a09a04b67bd0ae9e054f99cf470a039.png)

### System information (as much as possible)
Win10, 
firefox, edge 
Default system settings for fonts and display

**Comment by brianteeman** [comment]
Confirmed

**Comment by chmst** [comment]
A quick solution for the second point would be to shorten the text in the dropdown. But not sure if this helps always.

Speichern
... & Neu
... & Schließen

**Comment by brianteeman** [comment]
That would only resolve it for german and not any other language that has a longer text ;)

**Comment by chmst** [comment]
For many languages .. but it would be only dealing with symptoms. And not sure what happens with rtl languages.

**Comment by ggppdk** [comment]
> That would only resolve it for german and not any other language that has a longer text ;)

yes 
plus the text of 
3rd party custom buttons will not fit either

we have added override of the default CSS
which also seems to work with RTL

```css
#toolbar .buttons.btn-group > ul.dropdown-menu {
	min-width: 100% !important;
	width: auto !important;
	top: 100% !important;
	left: auto !important;
	transform: unset !important;
}
```

any better solution is welcome

**Comment by ghost** [comment]
closed as having Pull Request #22015
